.github/workflows/recipe-security-scanner.yml

```
@@ -0,0 +1,349 @@
+name: Recipe Security Scan
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'documentation/src/pages/recipes/data/recipes/**'
+
+concurrency:
+  group: scanner-${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  pull-requests: write
+  statuses: write
+
+jobs:
+  security-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@c6295a65d1254861815972266d5933fd6e532bdf # v2.11.1
+        with:
+          egress-policy: audit
+      
+      - name: Checkout PR
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Ensure jq available
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Find recipe files in PR
+        id: find_recipes
+        run: |
+          set -e
+          echo "Looking for recipe files in PR..."
+          
+          # Find all .yaml/.yml files in the recipes directory
+          RECIPE_FILES=$(find documentation/src/pages/recipes/data/recipes/ -name "*.yaml" -o -name "*.yml" 2>/dev/null || true)
+          
+          if [ -z "$RECIPE_FILES" ]; then
+            echo "No recipe files found in PR"
+            echo "has_recipes=false" >> "$GITHUB_OUTPUT"
+            echo "recipe_count=0" >> "$GITHUB_OUTPUT"
+          else
+            echo "Found recipe files:"
+            echo "$RECIPE_FILES"
+            RECIPE_COUNT=$(echo "$RECIPE_FILES" | wc -l)
+            echo "has_recipes=true" >> "$GITHUB_OUTPUT"
+            echo "recipe_count=$RECIPE_COUNT" >> "$GITHUB_OUTPUT"
+            
+            # Save recipe file paths for later steps
+            echo "$RECIPE_FILES" > "$RUNNER_TEMP/recipe_files.txt"
+          fi
+
+
+
+      - name: Set up Docker Buildx
+        if: steps.find_recipes.outputs.has_recipes == 'true'
+        uses: docker/setup-buildx-action@v3
```

double check but i believe this is what you need

```suggestion
        uses: docker/setup-buildx-action@1583c0f09d26c58c59d25b0eef29792b7ce99d9a
```